manager/manager/typeresolver.go

```diff
@@ -48,10 +53,12 @@ func NewTypeResolver() TypeResolver {
 	client.Timeout = timeout
 	ret.getter = util.NewHTTPClient(3, client, util.NewSleeper())
 	ret.maxUrls = maxURLImports
+	ret.re = regexp.MustCompile("github.com/(.*)/(.*)/(.*)/(.*):(.*)")
```

Comment:
Must we be this strict? It seems like the specification for it is github.com/<user>/<repo>/<path>:<version> where path could have arbitrarily deep nesting.